src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js

```diff
@@ -11,23 +11,33 @@ governing permissions and limitations under the License.
 */
 
 import { getNonce } from "../../dom-actions/dom";
-import { createElement, parseAnchor, removeElementById } from "../utils";
-import { TEXT_HTML } from "../../constants/contentType";
-import { assign } from "../../../../utils";
+import { parseAnchor } from "../utils";
+import { TEXT_HTML } from "../../../../constants/contentType";
+import { assign, includes, values } from "../../../../utils";
+import { createNode, removeNode } from "../../../../utils/dom";
+import { objectOf } from "../../../../utils/validation";
 import { PropositionEventType } from "../../../../constants/propositionEventType";
 import { INTERACT } from "../../../../constants/eventType";
 
 const ALLOY_MESSAGING_CONTAINER_ID = "alloy-messaging-container";
 const ALLOY_OVERLAY_CONTAINER_ID = "alloy-overlay-container";
 const ALLOY_IFRAME_ID = "alloy-content-iframe";
 
-const dismissMessage = () =>
-  [ALLOY_MESSAGING_CONTAINER_ID, ALLOY_OVERLAY_CONTAINER_ID].forEach(
-    removeElementById
-  );
+export const dismissMessage = () => {
```

Comment:
What is the reason you removed the `removeElementById` method?  I find the prior code for `dismissMessage` to be more readable.  Could we keep that method but modify it to use `removeNode`?

Reply:
I believe this code is quite readable. The `removeElementById` function might seem a bit redundant, especially when we already have `removeNode`. However, I don't have a strong preference, so I go ahead and include `removeElementById` back in the code.